src/GraphQL/Types/Collections/SchemaTypes.cs

```diff
@@ -656,7 +694,7 @@ private IGraphType AddTypeIfNotRegistered(Type type, TypeCollectionContext conte
                 else if (_builtInCustomScalars.TryGetValue(namedType, out var builtInCustomScalar))
                 {
                     foundType = builtInCustomScalar;
-                    AddType(foundType, _context);
+                    AddType(foundType, context);
```

Comment:
Why was this changed?  Bug?

Reply:
Accidentally. Now I puzzle over whether `_context` was used here on purpose. Awkward SchemaTypes code. I did not remember.

Reply:
"If it ain't broke, don't fix it".... unless you're sure.

Reply:
Yeah, I don't know if it was intentional or not.  As such, I personally wouldn't touch it, but I might add a comment.  I'll review the remainder of this PR and leave this change for you to decide.

Reply:
I reverted.